cadc-vos-server/src/main/java/ca/nrc/cadc/vos/server/web/action/FilesAction.java

```diff
@@ -0,0 +1,195 @@
+/*
+************************************************************************
+*******************  CANADIAN ASTRONOMY DATA CENTRE  *******************
+**************  CENTRE CANADIEN DE DONNÉES ASTRONOMIQUES  **************
+*
+*  (c) 2018.                            (c) 2018.
```

Comment:
Not picky with this but we need to get used to 2019 haha...

Reply:
Right... will fix